index.d.ts

```diff
@@ -7,3 +7,9 @@ export interface MercuriusExplainOptions {
 declare const mercuriusExplain: FastifyPluginAsync<MercuriusExplainOptions>
 
 export default mercuriusExplain
+
+export declare function explainGraphiQLPlugin(options?: { version?: string }): {
+  name: string,
+  umdUrl: string,
+  fetcherWrapper: string,
```

Comment:
can you use the `ExplainGraphiQLPluginReturnType` as return type of  `explainGraphiQLPlugin`?
Also can you create a type like `type explainGraphiQLPluginOptions = {
version?: string
}`